### PR TITLE
[FIX] connector_equipment: Mislabeled Variable

### PR DESCRIPTION
**Pull request by osi-scampbell**
Variable in method is misalabeled

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@osi-scampbell Can you add a test?

**Comment by osi-scampbell** [comment]
@max3903 Yes, adding tests now

File: connector_equipment/tests/test_connector_equipment.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import time
+from dateutil import relativedelta
```
Comment (bodedra) [inline connector_equipment/tests/test_connector_equipment.py]:
```suggestion

```

File: connector_equipment/tests/test_connector_equipment.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import datetime
```
Comment (bodedra) [inline connector_equipment/tests/test_connector_equipment.py]:
```suggestion

```

File: connector_equipment/tests/test_connector_equipment.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+from odoo import fields
```
Comment (bodedra) [inline connector_equipment/tests/test_connector_equipment.py]:
```suggestion

```

File: connector_equipment/tests/test_connector_equipment.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+from odoo.tests.common import TransactionCase, Form
```
Comment (bodedra) [inline connector_equipment/tests/test_connector_equipment.py]:
```suggestion
from odoo.tests.common import TransactionCase
```

File: connector_equipment/tests/test_connector_equipment.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class ConnectorEquipment(TransactionCase):
+
+    def setUp(self):
+        super(ConnectorEquipment, self).setUp()
+        self.backend_equipment = self.env['backend.equipment']
+        self.maintenance_equipment = self.env['maintenance.equipment']
+
```
Comment (bodedra) [inline connector_equipment/tests/test_connector_equipment.py:13]:
```suggestion
```

File: connector_equipment/tests/test_connector_equipment.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+            'serial_no': 'MT/127/18291015',
+            'model': 'NP355E5X',
+            'color': 3,
+            'backend_id': backend_01.id,
+            'managed': True
+
+        })
+
+        self.assertEqual(equipment_01.update_config(), True)
+
```
Comment (bodedra) [inline connector_equipment/tests/test_connector_equipment.py]:
```suggestion
```

**Review by bodedra — COMMENTED** [review]
minor corrections.

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Pull request merged**